config/rbac/role.yaml

```diff
@@ -54,16 +54,6 @@ rules:
   - get
   - list
   - watch
-- apiGroups:
-  - events.k8s.io
-  resources:
-  - events
-  verbs:
-  - create
```

Comment:
We don't create events?

Reply:
I don't think we are using the API Group `events.k8s.io` which was added for the deprecated event solution.